core/src/Nav/Node/Node.less

```diff
@@ -3,8 +3,8 @@
 .item {
   display: flex;
   align-items: baseline;
-  border-radius: 0 0 0 3px;
   user-select: none;
+  position: relative;
```

Comment:
Why are we setting relative positioning? Are we z-indexing children? Or positioning them?

Reply:
I can see that we are absolute positing the actions group. We should avoid this. I would recommend looking at flex or secondarily `vertical-align` in order to get the actions layout as always in the middle of the nav item text, even when that wraps and creates multiple lines.